src/FillCommand.php

```diff
@@ -26,9 +28,15 @@ public function handle()
             $resources = $resourceManager;
         }
 
+        if ($this->argument('resourceIds')) {
+            $resourceIds = collect($this->argument('resourceIds'));
```

Comment:
I personally don't see much value in making this a collection, all I do with it later is implode it